src/index.html

```diff
@@ -12,7 +12,49 @@
       href="./style.css"
     />
```

Comment:
The href attribute value './style.css' should not contain spaces. Ensure that there are no spaces in the link as per the checklist requirement: 'Don't use spaces in links'.